app/models/config.py

```diff
@@ -23,7 +23,7 @@ def default_data(token=None):
             "services": [Service().__dict__],
             "announce": Announcements().__dict__,
             "spam": Spam().__dict__,
-            "whitelisted_urls": []
+            "whitelist": Whitelist().__dict__
```

Comment:
`"whitelist": [Whitelist().__dict__]` since it's a dictionary. Same for blacklist.